Tests/PlayerTests/Player/PlaybackEngine/Internal/Cache/DBCache/DBCacheStorageTests.swift

```diff
@@ -0,0 +1,104 @@
+import GRDB
+@testable import Player
+import XCTest
+
+class DBCacheStorageTests: XCTestCase {
+	var dbQueue: DatabaseQueue!
+	var cacheStorage: DBCacheStorage!
+
+	static let cacheEntry1 = CacheEntry.mock(
```

Comment:
[suggestion] minor - I think these could be in a constants enum outside so we keep the class to only test/helper funcs and keep the actual class more concise, since most of the times these details are not relevant when reading the tests, just if we need to modify it 🙂

Reply:
yeah, agree. Because I only used them here I had not moved them out, but will do it.